tickerService.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"time"
+)
+
+type TickerService struct {
+	DeviceService DeviceService
+	measurement   chan Measurement
+	isRunning     bool
+}
+
+type Measurement struct {
+	Id    int
+	Value float64
+}
+
+func (ts *TickerService) Start() (started bool, error error) {
+	if !ts.isRunning {
+		deviceWithId, err := ts.DeviceService.Get(0, 0)
+		if err != nil {
+			return false, err
+		}
+		for _, device := range deviceWithId {
+			ticker := time.NewTicker(time.Duration(device.Interval) * time.Second)
```

Comment:
there is one common gotcha with golang for loop, goroutines and clojures. Let's discuss this f2f tomorrow but I will hint you now with a test - create ~10 devices, every device with different Value and see what happens :)